pandas-stubs/core/resample.pyi

```diff
@@ -62,9 +59,7 @@ _SeriesGroupByFuncArgs: TypeAlias = (
 )
 
 class Resampler(BaseGroupBy[NDFrameT]):
-    grouper: BinGrouper  # pyright: ignore[reportIncompatibleVariableOverride]  # pyrefly: ignore  # variance incompatibility
-    binner: DatetimeIndex | TimedeltaIndex | PeriodIndex
-    exclusions: frozenset[Hashable]
+    _grouper: BinGrouper
     ax: Index
```

Comment:
We should delete `_grouper` and `ax`

Reply:
I thought it was needed because my types were failing but it was instead the tests that depended on it, also removed grouper from the `BaseGroupBy` to align the subclass.